node/libs/roles/src/attester/messages/batch.rs

```diff
@@ -10,7 +10,7 @@ pub struct BatchNumber(pub u64);
 impl BatchNumber {
     /// Increment the batch number.
     pub fn next(&self) -> BatchNumber {
-        BatchNumber(self.0.checked_add(1).unwrap_or(0))
+        BatchNumber(self.0 + 1)
```

Comment:
`checked_add(1).unwrap()`. Overflows are only checked in dev builds. In release build it will just wrap.